src/OSPSuite.Presentation/Mappers/DiffItemToDiffItemDTOMapper.cs

```diff
@@ -86,7 +89,24 @@ private string presentWithDetails(string presentDetails)
 
       public void Visit(PropertyValueDiffItem propertyDiffItem)
       {
-         updateDiffItem(propertyDiffItem.FormattedValue1, propertyDiffItem.FormattedValue2, propertyDiffItem.PropertyName, objectNameFrom(propertyDiffItem), propertyDiffItem.CommonAncestor, itemIsMissing: false);
+         Func<PathElements> pathElementsMapper = () => mapPathElementsAsIEntity(propertyDiffItem.CommonAncestor);
+
+         if (propertyDiffItem.Object1.GetType() == propertyDiffItem.Object2.GetType() && propertyDiffItem.Object1.IsAnImplementationOf<PathAndValueEntity>())
```

Comment:
propertyDiffItem.Object1.GetType() == propertyDiffItem.Object2.GetType()
can this ever not be the case?